analysis/blacklist/blacklist.go

```diff
@@ -0,0 +1,85 @@
+package blacklist
+
+import (
+	bl "github.com/ocmdev/rita-blacklist2"
```

Comment:
At some point, we can probably just rename this rita-blacklist and get rid of the other one...